docs/Using-Pantheon/Accessing-Logs-Using-JSON-RPC.md

```diff
@@ -1,6 +1,8 @@
 # Accessing Logs Using Pantheon API
 
-Access logs using Pantheon API methods:
+You can subscribe to events, such as logs, by using either [RPC Pub/Sub over WebSockets](../Pantheon-API/RPC-PubSub.md) or filters over HTTP.  
```

Comment:
Similar comment to above: 
Subscribe to events, such as logs, using either filters over HTTP or RPC Pub/Sub....